### PR TITLE
Add doc for tiflash scheduler limit parameters

### PR DESCRIPTION
**Pull request by xzhangxian1008**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions))

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -244,6 +244,15 @@ delta_index_cache_size = 0
     ## 从 v7.4.0 引入，表示是否开启 TiFlash 资源管控功能。当设置为 true 时，TiFlash 会使用 Pipeline Model 执行模型。
     enable_resource_control = true
 
+    ## 从 v6.0.0 引入，表示当前 resource group 能够使用的最多线程数。如果超出，当前 query 会被放入等待队列。
+    task_scheduler_thread_soft_limit = 5000
+
+    ## 从 v6.0.0 引入，表示全局线程数的最大值。如果超出，当前 query 会被放入等待队列，直至下一次的调度。
```
[the remark this answers is not in the document]
Comment (windtalker) [reply tiflash/tiflash-configuration.md:250]: 而且超过这个值会报错而不是加入等待队列

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -244,6 +244,15 @@ delta_index_cache_size = 0
...
+    task_scheduler_thread_hard_limit = 10000
+
+    ## 从 v6.4.0 引入，表示当前 tiflash 节点最多可以同时运行多少个 query。在 v6.5 版本中，该参数的默认值是 vcpu 数量的四分之一。用户在设置时，该参数的范围在 0 到五十倍 vcpu 之间，如果参数值超过这个范围，那么会被自动调整为 vcpu 数量的四分之一。在 v7.1 版本中，该参数的默认值是 vcpu 数量的四分之一。在 v7.5 和 v8.1 版本中，该参数的默认值是两倍的 vcpu 数量。
```
Comment (windtalker) [inline tiflash/tiflash-configuration.md:253]: 这个是要cherry pick回各个branch的吧，我感觉没必要在这里把所有版本都写出来？

Comment (xzhangxian1008) [reply tiflash/tiflash-configuration.md]: > 这个是要cherry pick回各个branch的吧，我感觉没必要在这里把所有版本都写出来？

改好了

**Pull request closed without merging**